File: Java/src/main/java/at/nagarro/exercises/bank/Customer.java
package at.nagarro.exercises.bank;

import java.time.LocalDate;

public class Customer {
    public String firstName;
    public String surName;
    public LocalDate birthday;
    public Adress adress;

    public String fullName() {
        return firstName.toLowerCase() + " " + surName;
    }

    public String email() {
        return firstName.toLowerCase() + "." + surName.toLowerCase() + "@mybank.com";
    }

}

File: TypeScript/src/banking/customer.ts
import {Adress} from "./adress";

export class Customer {
    firstName: string;
    surName: string;
    birthday: Date;
    adress: Adress;

    fullName(): string {
        return `${this.firstName.toLowerCase()} ${this.surName}`;
    }

    email(): string {
        return this.firstName.toLowerCase() + "." + this.surName.toLowerCase() + "@mybank.com";
    }

}